### PR TITLE
feat: Countries - Backend

### LINKED ISSUE #260: Countries - Backend
Model in the backend:

- only column is country

Endpoints:

- seed the countries database with all valid countries (make sure only a few countries)

- POST /countries - adds country to table, 

Payload: [{
"country_name": "name"
},
...]
- GET /countries -  get all countries in DB 

select country: canada, us, .., random_country that doesn't legally exist

### PR DESCRIPTION
**Pull request by Shehryar21**
closes #260 

Includes seeding logic, get all, and post country endpoint

sample request to add countries:

image

### CODE REVIEW

File: restapi/authors.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (api api) CreateAuthors(w http.ResponseWriter, r *http.Request) render.Rend
 	return rest.MsgStatusOK("Authors added successfully")
 }
 
-func (api api) ReturnAllCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
+func (api api) ReturnAllUsedCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
```
Comment (nimwijetunga) [inline restapi/authors.go]: This name is kind of misleading, lets use `ReturnAuthorOriginCountries` or something along those lines

File: restapi/countries.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		return rest.ErrInternal(api.logger, err)
+	}
+	names := make([]string, len(countries))
+	for i, t := range countries {
+		names[i] = t.Name
+	}
+	return rest.JSONStatusOK(names)
+}
+
+func (api api) AddCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
```
Comment (nimwijetunga) [inline restapi/countries.go:40]: Can you document a sample payload here so frontend can use it as a reference when they call this endpoint?

**Review by nimwijetunga — APPROVED** [review]
LGTM, please make sure you test that these endpoints work as expected before merging

File: restapi/countries.go (hunk 2 above)
Comment (Shehryar21) [reply restapi/countries.go:40]: like in the code as a comment?

**Pull request merged**